com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/CopybookServiceTest.java

```diff
@@ -151,9 +141,8 @@ public void findCopybookWithDatasetFilteringNegativeTest() {
   public void findCopybookWithWrongFolderStructure() {
     assertNull(
         copybookService.findCopybook(
-            CPY_NAME_WITHOUT_EXT,
-            (String) configurationSettingsStorable.getProfiles(),
-            Collections.singletonList("HLQLF02.DSNAME1")));
+                CPY_NAME_WITHOUT_EXT,
+            Collections.singletonList(PROFILE_NAME + filesystemSeparator() + "HLQLF02.DSNAME1")));
```

Comment:
Since we remove the profile concept cause is built into the path I prefer remove it as well from tests